tools/perspective-scripts/publish.mjs

```diff
@@ -221,11 +247,25 @@ if (proceed.toLowerCase() === "y") {
                 }
             );
 
-            // Write out the zip file
-            await fs.appendFile(
-                `${dist_folder}/${artifact.name}.zip`,
-                Buffer.from(download.data)
-            );
+            await octokit.request({
+                method: "POST",
+                url: release.data.upload_url,
+                owner: "finos",
+                repo: "perspective",
+                data: download.data,
+                name: `${artifact.name}.zip`,
+                headers: {
+                    "X-GitHub-Api-Version": "2022-11-28",
+                },
+            });
+
+            if (!artifact.name.includes("pyodide")) {
+                // Write out the zip file
+                await fs.appendFile(
+                    `${dist_folder}/${artifact.name}.zip`,
+                    Buffer.from(download.data)
+                );
+            }
         })
     );
 
```

Comment:
Need the same checks for `pyodide` everywhere `python_dist_folders` is iterated over